website/utils/renderer.ts

```diff
@@ -7,7 +9,15 @@ export const renderStatic = async (callback: () => string) => {
     throw new Error('did you forget to return html from renderToString?');
   }
 
-  const { ids, css } = extractCritical(html);
+  const { extractCritical: websiteExtractCritical } = createEmotionServer(
+    cache,
+  );
+
+  const { ids: websiteIds, css: websiteCss } = websiteExtractCritical(html);
+  const { ids: lgIds, css: lgCss } = lgExtractCritical(html);
```

Comment:
This could use a detailed comment explaining the the necessity of these separate operations. this could be illuminating for a future maintainer

Reply:
👍